coral-component-table/src/scripts/Table.js

```diff
@@ -238,8 +238,14 @@ const Table = Decorator(class extends BaseComponent(HTMLTableElement) {
         this._elements.table.appendChild(body);
         this.items._container = body;
 
+
         // To init the body observer
         body.setAttribute('_observe', 'on');
+
+        const rows = getRows([body]);
+        // Use the first column as selection column
+        // @todo check whether whole state needs to be synced or not.
+        rows.forEach(row => this._toggleSelectionCheckbox(row));
```

Comment:
Do we need to do more here, like what we do in `_onBodyContentChanged` event handler? https://github.com/adobe/coral-spectrum/blob/07e3dcc432661cba23d3ffba7badcb68b6d69f50/coral-component-table/src/scripts/Table.js#L1370-L1409

Reply:
Hi @majornista ,
I have corrected the code to call _onBodyContentChanged when table body is inserted.